v1/src/lib.rs

```diff
@@ -463,15 +583,30 @@ impl Storage for InfluxDbStorage {
         .add_field("base64", base64)
         .add_field("value", strvalue);
 
-        debug!("Put {:?} with Influx query: {:?}", measurement, query);
-        if let Err(e) = self.client.query(&query).await {
-            bail!(
-                "Failed to put Value for {:?} in InfluxDb storage : {}",
-                measurement,
-                e
-            )
-        } else {
-            Ok(StorageInsertionResult::Inserted)
+        match &self.put_batch_tx {
+            None => {
+                // not batched - send query now
+                debug!("Put {:?} with Influx query: {:?}", measurement, query);
+                if let Err(e) = self.client.query(&query).await {
+                    bail!(
+                        "Failed to put Value for {:?} in InfluxDb storage : {}",
+                        measurement,
+                        e
+                    )
+                } else {
+                    Ok(StorageInsertionResult::Inserted)
+                }
+            }
+            Some(sender) => {
+                let put = Put { query, measurement };
+                if let Err(e) = sender.send(put) {
+                    bail!("Failed to send to batch queue for InfluxDb storage : {}", e)
+                } else {
+                    // assume success
+                    // TODO - add pending status
+                    Ok(StorageInsertionResult::Inserted)
```

Comment:
I'm not 100% sure about returning a`StorageInsertionResult::Inserted` as the insertion could fail inside the batch, but it has been reported to zenoh as inserted. This will likely have implications inside replication and storage alignment. 

The suggestion to add a `Pending` status could work, however then we will also have to add a `Completed` status that will have to be signaled to zenoh that all of those Puts have been successful as a group. 
This may require an additional ID per `Put` when batching, and upon successful write, the ID's of the batch would have to be signaled back to zenoh.

@JEnoch, What do we think about adding a `Pending` and `Complete` system ? 
it adds a little bit of complication, and i'd like to make sure we don't break alignment due to batching.